### ISSUE TITLE
it does not work

### ISSUE THREAD

**Issue by alexlii1971** [alexc7018/bbpress-ajax-replies#2]
Hello,

Great work, and I found it does not work, and did find setup page at backend, anyway to resolve?

Thanks

Alex

**Comment by crlucasdias** [comment]
I have problems too.
Did u found a solution? @alexlii1971 @aliso

**Comment by alexlii1971** [comment]
@fourtest, NO, actually, I give up BBpress, and use Anspress now:https://wordpress.org/plugins/anspress-question-answer/
